setup.cfg

```diff
@@ -46,18 +46,18 @@ exclude =
 
 [options.extras_require]
 dev =
```

Comment:
we probably don't want to pin to these specific versions in perpetuity, do we?

Reply:
What about setting these as minimums instead of pinning?  I don't love pinning to specific versions but have had many experiences with Python where newer versions break things in unexpected ways.

Reply:
Changed to >= for most modules